src/util/object_factory_parameters.h

```diff
@@ -70,6 +71,14 @@ struct object_factory_parameterst
   /// Force string content to be ASCII printable characters when set to true.
   bool string_printable = false;
 
+  /// Force numerical primitive inputs to fall within the bounds
+  struct numeric_input_limitst
+  {
+    optionalt<int> lower;
+    optionalt<int> upper;
```

Comment:
Is `int` really the right type? I'd use `mp_integer` here as the type of the system JBMC is being compiled on is unrelated to the type that you want to talk about.

Reply:
Good point, done!